app/config/config.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const localEnv = _loadLocalEnv(),
+    env = process.env.NODE_ENV || 'development';
+
+
+/**
+ * @property {string} HOST
+ * @property {string} PORT
+ * @property {string} MONGODB_URL
+ */
+const config = {
+
+    development: {
+        HOST: localEnv.HOST || '188.166.63.248',
+        PORT: localEnv.PORT || '9030',
+
+        MONGODB_URL: localEnv.MONGODB_URL || 'mongodb://localhost:27017/differ-canvas'
+    },
+
+    test: {
+        HOST: '127.0.0.1',
+        PORT: '3000',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas_test_integration'
+    },
+
+    production: {
+        HOST: '127.0.0.1',
+        PORT: '9010',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas'
+    }
+};
+
+
+module.exports = config[env];
+
+
+/**
+ * @type function
```

Comment:
validateBody can be a middleware that uses body directly too instead of passing req.body. A middleware that accepts parameters to test.

const validateBody = (parameters) => {
   return (req, res, next) => {
     // do validation
    }
}